src/graphql/data/apollo.ts

```diff
@@ -11,5 +12,13 @@ export const apolloClient = new ApolloClient({
     'Content-Type': 'application/json',
     Origin: 'https://app.uniswap.org',
   },
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache({
+    typePolicies: {
+      Query: {
+        fields: {
+          nftBalances: relayStylePagination(),
```

Comment:
do you need to pass in any `keyArgs` here for caching? Depends on if/ how you do caching, but on mobile i had issues where the entries between wallets were referencing the same query cache unless i added `ownerAddress`.

Reply:
Try now, I've updated the fetch policy and able to switch wallets with no issues